pkg/testcase/ingressdns.go

```diff
@@ -59,3 +61,18 @@ func TestDnsAccess(deleteWorkload bool) {
 		Expect(err).NotTo(HaveOccurred(), "dnsutils manifest not deleted")
 	}
 }
+
+func TestIngressDualStack(testinfo map[string]string) {
+	ingressIPs, err := shared.FetchIngressIP(testinfo["namespace"])
+	Expect(err).NotTo(HaveOccurred(), "Ingress ip is not returned")
+
+	for _, ingressIP := range ingressIPs {
+		if strings.Contains(ingressIP,":") {
+			ingressIP = IPv6IPEncloser(ingressIP)
+		}
+		err := assert.ValidateOnNode(shared.BastionIP,
+			"curl -sL -H 'Host: test1.com' http://"+ ingressIP +"/name.html",
+			testinfo["expected"])
+		Expect(err).NotTo(HaveOccurred(), err)
+	}
```

Comment:
Same comment as above -- basically: fail on first or fail after all are checked?

Reply:
Ideally should fail on first but I am on the fence on how we want to go about doing it. If we continue even on fail, there is currently no way to run tests with specific data unless we want to separate the tests itself and we have to re-run the test anyways if anything fails...

Reply:
We decided internally that this is fine as is!